sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -2336,8 +2331,18 @@ inline pi_result piKernelGetInfo(pi_kernel Kernel, pi_kernel_info ParamName,
     break;
   }
   case PI_KERNEL_INFO_NUM_ARGS: {
-    UrParamName = UR_KERNEL_INFO_NUM_ARGS;
-    break;
+    size_t NumArgs = 0;

```

Comment:
why do we need this change? this seems like something it should go in a completely separate patch so in case there are regressions it is easy to identify.

Reply:
without this one of the fixes we're pulling in causes a regression

Reply:
This changes was temporarily until this [PR](https://github.com/intel/llvm/pull/11794) is merged. This changes have been reverted and the PR was rebased on recent SYCL branch.